src/helpers/hotkeysManager.js

```diff
@@ -259,8 +268,11 @@ WebViewer(...)
         const { scrollHeight, clientHeight } = scrollViewElement;
         const reachedTop = scrollViewElement.scrollTop === 0;
         const currPageNumber = core.getCurrentPage();
-        if (reachedTop && currPageNumber > 1) {
-          core.setCurrentPage(currPageNumber - 1);
+        if (!this.isContinuous() && reachedTop && currPageNumber > 1) {
```

Comment:
Do we actually need the `isContinuous` function? If it's in continuous mode then `reachedTop` will be false when the current page isn't the first.

Reply:
I guess we don't need this. I'll remove it